cvl/internal/yparser/yparser.go

```diff
@@ -278,9 +373,7 @@ type YParserLeafValue struct {
 }
 
 type YParser struct {
-	//ctx *YParserCtx    //Parser context
-	root      *YParserNode //Top evel root for validation
-	operation string       //Edit operation
+	dummy string // No members?
```

Comment:
golang allows empty struct type. You can avoid dummy fields. Doesn't libyang3 need a struct_ly_ctx?

Reply:
Yeah, a ly_ctx is needed, but the original author made it global rather than a struct/class member for some reason.  I wasn't trying to rearchitect for proper design :)

I'll remove the dummy member.

Reply:
done